x-pack/plugins/session_view/public/components/SessionViewDetailPanel/index.tsx

```diff
@@ -147,19 +147,26 @@ export const SessionViewDetailPanel = ({
       onAnimationEnd={handleAnimationEnd}
     >
       {renderSelectedProcessCommandDetail()}
-      <EuiTitle size="s">
-        <span>
-          <FormattedMessage id="xpack.sessionView.sessionDetail" defaultMessage="Session detail" />
-        </span>
-      </EuiTitle>
-      {/* Add session detail */}
+      <div data-test-subj="sessionViewDetailPanelSessionDetail">
+        <EuiTitle size="s">
+          <span>
+            <FormattedMessage
+              id="xpack.sessionView.sessionDetail"
+              defaultMessage="Session detail"
+            />
+          </span>
+        </EuiTitle>
+        {/* Add session detail */}
+      </div>
       <EuiSpacer size="xxl" />
-      <EuiTitle size="s">
-        <span>
-          <FormattedMessage id="xpack.sessionView.serverDetail" defaultMessage="Server detail" />
-        </span>
-      </EuiTitle>
-      {/* Add server detail */}
+      <div data-test-subj="sessionViewDetailPanelServerDetail">
+        <EuiTitle size="s">
+          <span>
+            <FormattedMessage id="xpack.sessionView.serverDetail" defaultMessage="Server detail" />
```

Comment:
nitpick: let's make the newlines consistent with lines 153-156

Reply:
i wanted to but the prettier wouldn't let me lol 
![image](https://user-images.githubusercontent.com/16872649/144107758-4996f1eb-18d6-46e5-97ae-8209a8ddd321.png)